src/components/Page/components/Header/Header.tsx

```diff
@@ -123,12 +128,19 @@ class Header extends React.PureComponent<ComposedProps, State> {
         </div>
       ) : null;
 
+    const subtitleMarkup = subtitle ? (
+      <div className={styles.Subtitle}>
+        <Heading element="h2">{subtitle}</Heading>
+      </div>
+    ) : null;
+
     const titleMarkup = (
       <div className={styles.Title}>
-        <div className={styles.DisplayTextWrapper}>
```

Comment:
I removed the `className` since it wasn't actually being used in any of the stylesheets.